package.json

```diff
@@ -148,25 +160,36 @@
     "@types/ua-parser-js": "^0.7.36",
     "@types/uuid": "^8.3.4",
     "@types/wcag-contrast": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
+    "@typescript-eslint/eslint-plugin": "^5.58.0",
+    "@typescript-eslint/parser": "^5.58.0",
+    "@vitejs/plugin-react": "^3.1.0",
+    "babel-plugin-macros": "^3.1.0",
     "compression-webpack-plugin": "^6.1.1",
```

Comment:
"react-app-rewire-styled-components"
"react-app-rewired"
"compression-webpack-plugin"
"source-map-explorer"

Please check these deps. They may not be in need any more